src/amo/components/UserProfile/index.js

```diff
@@ -206,28 +201,18 @@ export class UserProfileBase extends React.Component<InternalProps> {
       >
         <ul>
           {reviews.map((review) => {
-            const reviewBodySanitized = sanitizeHTML(nl2br(review.body), [
-              'br',
-            ]);
+            const byLine = (
+              <Link
+                title={i18n.gettext('Browse the reviews for this add-on')}
+                to={`/addon/${review.addonSlug}/reviews/`}
+              >
+                {i18n.moment(review.created).fromNow()}
+              </Link>
+            );
 
             return (
               <li key={String(review.id)}>
-                <div className="AddonReviewListItem">
-                  <p
-                    className="AddonReviewListItem-body"
-                    // eslint-disable-next-line react/no-danger
-                    dangerouslySetInnerHTML={reviewBodySanitized}
-                  />
-                  <div className="AddonReviewListItem-byline">
-                    <UserRating styleSize="small" review={review} readOnly />
-                    <Link
-                      title={i18n.gettext('Browse the reviews for this add-on')}
-                      to={`/addon/${review.addonSlug}/reviews/`}
-                    >
-                      {i18n.moment(review.created).fromNow()}
-                    </Link>
-                  </div>
-                </div>
+                <UserReview review={review} byLine={byLine} />
```

Comment:
I tried this and it actually works quite well:

```diff
diff --git a/src/amo/components/UserProfile/index.js b/src/amo/components/UserProfile/index.js
index 6d0f2e8a8..11e48d01c 100644
--- a/src/amo/components/UserProfile/index.js
+++ b/src/amo/components/UserProfile/index.js
@@ -7,6 +7,7 @@ import { compose } from 'redux';
 
 import { fetchUserReviews } from 'amo/actions/reviews';
 import AddonsByAuthorsCard from 'amo/components/AddonsByAuthorsCard';
+import AddonReviewListItem from 'amo/components/AddonReviewListItem';
 import Link from 'amo/components/Link';
 import NotFound from 'amo/components/ErrorPage/NotFound';
 import ReportUserAbuse from 'amo/components/ReportUserAbuse';
@@ -212,7 +213,7 @@ export class UserProfileBase extends React.Component<InternalProps> {
 
             return (
               <li key={String(review.id)}>
-                <UserReview review={review} byLine={byLine} />
+                <AddonReviewListItem review={review} location={location} />
               </li>
             );
           })}
```

- Editing mostly works. I had to refresh the page to see it. Selecting stars isn't working.
- Developer responses are shown which is nice because it allows the developer to clarify things.
- The *flag review* menu is there but we could add a prop to hide it. If we were to show user reviews to *other* site users then the *flag review* menu would make sense here.
- This makes the byline consistent but I'm not sure if that's what we want.

Just a thought. It could be done in future issues.

Reply:
Oh, actually the *flag review* menu makes perfect sense here for developer replies.

Reply:
mmh, so you'd like to have write access to the reviews on the user profile page?

Reply:
Sure, I don't see why not. The feature has been requested before. And it would keep things consistent across the site.